demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -232,12 +232,21 @@ interface Toggle {
 }
 
 interface TranscriptSegment {
-  content: string;
+  contentSpan: HTMLSpanElement,
   attendee: Attendee;
   startTimeMs: number;
   endTimeMs: number;
 }
 
+interface TranscriptionStreamParams {
+  contentIdentificationType?: 'PII' | 'PHI';
+  contentRedactionType?: 'PII';
+  enablePartialResultsStability?: boolean;
+  partialResultsStability?: string;
```

Comment:
Nit, should `partialResultsStability` also be a literal type, e.g. `'high' | 'medium' | 'low'`?